styles.css

```diff
@@ -110,11 +114,13 @@ img {
 
 .no-bullets {
 list-style-type: none; 
+margin: 4px 10px;
 }
 
 .buttontext {
-  background-color: rgba(176, 146, 201, 0.7);
+  background-color: rgba(164, 130, 192, 0.7);
```

Comment:
May you could choose another color, which has a higher constrast. Check this out [Color Picker](https://imagecolorpicker.com/color-code/2596be)

Reply:
haha i know, will be my task now - I chose these colors at the very beginning and didnt check the contrast calculater till now 🙈